NSS-Day-Cohort-55/tabloidfullstack-articulate-proficient-isopods
# Cc 21 edit user

TESTING INSTRUCTIONS:

Navigate to users
Click Edit
Click "Change" an alert should appear asking if you're sure. Click "Yes" to change the user's type, click "Cancel" to close the alert. 
The user should now have a different User Type
Click "Back" to go back to the user list.
